### ISSUE TITLE
Inconsistent behavior of "View Source" actions

### ISSUE THREAD

**Issue by SimonMarquis** [JakeWharton/SdkSearch#41]
View source action will lead to 2 different places depending on the button pressed:
- in the list view (main screen): the `View source` button of the overflow menu item will open a ChromeCustomTab
- in the details view (ChromeCustomTab): the "View * source" button in the ActionBar will open a regular ACTION_VIEW intent, and open whatever browser you have on your device

I'm working on the 2nd "View Source" action to load the uri into a ChromeCustomTab as well.

**Comment by JakeWharton** [comment]
What's the behavior when you click "View Source" and it launches a custom tab from a custom tab? Does it overlay or does it redirect the current one?

**Comment by SimonMarquis** [comment]
It does create another Activity (CustomTab) on top of the current one (back button returns to the initial CustomTab).
Unfortunately, there is no API to redirect the user from within a CustomTab...

**Comment by JakeWharton** [comment]
Yeah I briefly went looking for one before deciding on the ACTION_VIEW intent. I'll try out the overlaying one in your PR and see how it feels.

**Comment by SimonMarquis** [comment]
![2018-01-17_19-26-03](https://user-images.githubusercontent.com/1921278/35059732-6d0e7036-fbbc-11e7-935a-58996f5075cf.gif)
